recipe/build.sh

```diff
@@ -9,11 +9,30 @@ if [[ "${target_platform}" == osx-* ]]; then
   export CXXFLAGS="${CXXFLAGS} -DTARGET_OS_OSX=1"
 fi
 
+# For builds that are cross-compiled (aarch64), we need to build a bootstrap system first.
+# https://www.erlang.org/doc/installation_guide/install-cross#Build-and-Install-Procedure_Building-With-configuremake-Directly_Building-a-Bootstrap-System
+if [[ "${CONDA_BUILD_CROSS_COMPILATION}" -eq 1 ]]; then
+
+  BOOTSTRAP_PREFIX="${PREFIX}/../bootstrap"
+  CFLAGS= LDFLAGS= ./configure \
+      --enable-bootstrap-only \
+      --host="${CONDA_TOOLCHAIN_BUILD}" \
+      --prefix="${BOOTSTRAP_PREFIX}"
+
+  make -j $CPU_COUNT
+  make install
+
+  export PATH="${BOOTSTRAP_PREFIX}/bin:${PATH}"
+  export LIBRARY_PATH="${BOOTSTRAP_PREFIX}/lib:${LIBRARY_PATH}"
+  export LD_LIBRARY_PATH="${BOOTSTRAP_PREFIX}/lib:${LD_LIBRARY_PATH}"
```

Comment:
Adding the `$PATH` seems ok. Would suggest dropping the rest though as we may not want to link to non-host compatible libraries by accident